force-app/main/default/classes/DonationHistoryController_TEST.cls

```diff
@@ -137,7 +137,7 @@ public class DonationHistoryController_TEST {
             Test.startTest();
             System.runAs(u){
                 List<DonationHistoryController.DonationHistoryView> donationHistory = 
-                DonationHistoryController.getDonationHistory(testContact.Id); 
+                DonationHistoryController.getDonationHistory(testContact.Id, 0); 
             }
 	        Test.stopTest();
         } catch(Exception e){
```

Comment:
dont we need a new test for the new method?

Reply:
change the implementation of the test so that it can correctly evaluate the new response with the data of the number of records and the list of donations